<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>pianWHOA</class>
 <widget class="QMainWindow" name="pianWHOA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>875</width>
    <height>506</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PianWHOA!</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true">background-color:black;
</string>
   </property>
   <widget class="QPushButton" name="aButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(255,182,193);
}

QPushButton:checked{
background: rgb(255,182,193);
}
    
if (aButton-&gt;isOn())
       btnAbort-&gt;setPaletteBackgroundColor( QColor(255,182,193) );
else
       btnAbort-&gt;setPaletteBackgroundColor(
    this-&gt;paletteBackgroundColor() );
</string>
    </property>
    <property name="text">
     <string>A</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="sButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(205,16,118);
}
QPushButton:checked{
background: rgb(205,16,118);
}</string>
    </property>
    <property name="text">
     <string>S</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="dButton">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(205,55,0);
}
QPushButton:checked{
background: rgb(205,55,0);
}</string>
    </property>
    <property name="text">
     <string>D</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="fButton">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(255,128,0);
}
QPushButton:checked{
background: rgb(255,128,0);
}</string>
    </property>
    <property name="text">
     <string>F</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="gButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(255,255,0);
}
QPushButton:checked{
background: rgb(255,255,0);
}</string>
    </property>
    <property name="text">
     <string>G</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="hButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(113,198,113);
}
QPushButton:checked{
background: rgb(113,198,113);
}</string>
    </property>
    <property name="text">
     <string>H</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="jButton">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(0,100,0);
}
QPushButton:checked{
background: rgb(0,100,0);
}</string>
    </property>
    <property name="text">
     <string>J</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="kButton">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(0,255,255);
}
QPushButton:checked{
background: rgb(0,255,255);
}</string>
    </property>
    <property name="text">
     <string>K</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="wButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(255,130,171);
}
QPushButton:checked{
background: rgb(255,130,171);
}</string>
    </property>
    <property name="text">
     <string>W</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="eButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(255,0,0);
}
QPushButton:checked{
background: rgb(255,0,0);
}</string>
    </property>
    <property name="text">
     <string>E</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="tButton">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(255,193,37);
}
QPushButton:checked{
background: rgb(255,193,37);
}</string>
    </property>
    <property name="text">
     <string>T</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="yButton">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(255,215,0);
}
QPushButton:checked{
background: rgb(255,215,0);
}</string>
    </property>
    <property name="text">
     <string>Y</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="uButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(0,139,69);
}
QPushButton:checked{
background: rgb(0,139,69);
}</string>
    </property>
    <property name="text">
     <string>U</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
QPushButton{
background:red;
border: 1px solid black;
color: white;
}
</string>
    </property>
    <property name="text">
     <string>MAIN MENU</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lButton">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(30,144,255);
}
QPushButton:checked{
background: rgb(30,144,255);
}</string>
    </property>
    <property name="text">
     <string>L</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="semiButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(255,0,255);
}
QPushButton:checked{
background: rgb(255,0,255);
}</string>
    </property>
    <property name="text">
     <string>;</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="apostButton">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>110</y>
      <width>61</width>
      <height>271</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:white;
border: 1px solid black;
}
QPushButton:pressed{
background: rgb(75,0,130);
}
QPushButton:checked{
background: rgb(75,0,130);
}</string>
    </property>
    <property name="text">
     <string>'</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="oButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(122,197,205);
}
QPushButton:checked{
background: rgb(122,197,205);
}</string>
    </property>
    <property name="text">
     <string>O</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>110</y>
      <width>41</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:black;
border: 1px solid black;
color:white;
}
QPushButton:pressed{
background: rgb(16,78,139);
}
QPushButton:checked{
background: rgb(16,78,139);
}</string>
    </property>
    <property name="text">
     <string>P</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>191</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>36</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:yellow;</string>
    </property>
    <property name="text">
     <string>PianWHOA!</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>pianWHOA</receiver>
   <slot>tutor()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>61</y>
    </hint>
    <hint type="destinationlabel">
     <x>296</x>
     <y>55</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>pianWHOA</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>65</y>
    </hint>
    <hint type="destinationlabel">
     <x>27</x>
     <y>151</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>tutor()</slot>
 </slots>
</ui>
